index.html

```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
 <section id="past-committee">
   <h2>Past Committee Members</h2>
   <ul>
+    <li>Ben Tillman (president 2022-2023)</li>
+    <li>Michael Yin (secretary 2021-2023)</li>
+    <li>Mark Haylock (general member 2021-2023)</li>
+    <li>Sarah Hay (general member 2022-2023)</li>
```

Comment:
This should be (general member 2021-2022, president 2023)

Reply:
yes, ditto.